tests/integration/cluster-users/k3d-cluster-users.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+set -e
+
+KYMA_PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../../../ &> /dev/null && pwd )"
+
+function host::os() {
+  local host_os
+  case "$(uname -s)" in
+  Linux)
+    host_os=linux
+    ;;
+  *)
+    echo >&2 -e "Unsupported host OS. Must be Linux"
+    exit 1
+    ;;
+  esac
+  echo "${host_os}"
+}
+
+function install::docker() {
+  if ! command -v docker &>/dev/null; then
+    curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+    echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian \
+    $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list >/dev/null
+    sudo apt-get update
+    sudo apt-get install docker-ce docker-ce-cli containerd.io -y
+  fi
+
+}
+
+function install::docker_compose() {
+  if ! command -v docker-compose &>/dev/null; then
+    sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
Is it related to the docker version? `1.29.2` may become obsolete. It would be nice to have it version-independent, e.g: always fetch `latest` or `stable` version for example

Reply:
`1.29.2` is the current latest release. I'd rather use specific and tested version to avoid getting breaking changes, especially that for docker-compose the 2.0 is coming soon.

Reply:
according to [docker-compose compatibility matrix](https://docs.docker.com/compose/compose-file/#compose-and-docker-compatibility-matrix) bumping docker should not cause problems.